WordPress/Classes/ViewRelated/Post/PostListEditorPresenter.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+
+/// Handle a user tapping a post in the post list. If an autosave revision is available, give the
+/// user the option through a dialog alert to load the autosave (or just load the regular post) into
+/// the editor.
+/// Analytics are also tracked.
+enum PostListEditorPresenter {
```

Comment:
I'm curious about the usage of `enum`. It can be misleading as I would expect some `cases` to go with it. But this one doesn't. I wonder if a `class` or `struct` would be better. 🤔 

If we are going to create an isolated unit test for this in the future, we'd most probably be getting rid of all the `static` functions in here so we can inject some (possibly mocked) dependencies inside it. So an `enum` is probably the least useful option. 

What do you think?

Reply:
I have been using enumerations for name-spacing lately because they cannot be accidentally instantiated (unlike classes or structures). I agree it can look a bit strange.

I also agree that with static methods, dependency injection is less straightforward. We could either switch to instance methods or make them "pure" functions with their dependencies injected.

In short, I favor your suggestion as it keeps things easier to maintain :)

Reply:
Updated to use a `struct`. Thanks for reminding me about unit tests here, if you'd like I can address these now or leave them for a future issue.